packages/lisk-p2p/src/helpers.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { VError } from 'verror';
+import { Peer } from './peer';
+
+export interface PeerReturnType {
+	readonly options: PeerOptions;
+	readonly peers: ReadonlyArray<Peer>;
+}
+export interface PeerOptions {
+	readonly [key: string]: string | number;
+}
+/* tslint:disable: readonly-keyword*/
+interface Histogram {
+	[key: number]: number;
+}
+interface HistogramValues {
+	height: number;
+	histogram: Histogram;
+	max: number;
+}
+/* tslint:enable: readonly-keyword */
+export const selectPeers = (
+	peers: ReadonlyArray<Peer>,
+	options: PeerOptions = { blockHeight: 0 },
+	numOfPeers?: number,
+): PeerReturnType => {
+	const filteredPeers = peers.filter(
+		// Remove unreachable peers or heights below last block height
+		(peer: Peer) => peer.Height >= options.blockHeight,
+	);
+
+	if (filteredPeers.length === 0) {
+		const optionsTemp = { ...options, blockHeight: 0 };
+
+		return { options: optionsTemp, peers: [] };
+	}
+
+	// Order peers by descending height
+	const sortedPeers = filteredPeers.sort((a, b) => b.Height - a.Height);
+
+	const aggregation = 2;
+	const defaultValue: HistogramValues = { height: 0, histogram: {}, max: -1 };
```

Comment:
here also, no need to define here?

Reply:
earlier it was giving me lint error without defining it, restarted TS and it works now so I will remove